cloudant-client/src/test/java/com/cloudant/tests/SearchTests.java

```diff
@@ -273,5 +273,14 @@ public void escapedAmpersandTest() {
         escapingTest("class:mammal%26test%26escaping", "class:mammal&test&escaping");
     }
 
+    @Test
+    public void searchOptionsStaleTest() {
```

Comment:
I don't understand how this test relates to the view changes. Has the `stale` parameter for `_search` also changed? It didn't appear so from the docs. Is this just filling a test gap? If so perhaps it could move into another commit just for clarity?

Reply:
Thanks for spotting, this was something left over from an earlier change, removed in 49baa12